Sources/BraintreeDropIn/BTDropInUICustomization.m

```diff
@@ -0,0 +1,79 @@
+#import "BTDropInUICustomization.h"
+
+#ifdef COCOAPODS
+#import <BraintreeDropIn/UIColor+BTUIK.h>
+#else
+#import <BraintreeUIKit/UIColor+BTUIK.h>
+#endif
+
+@implementation BTDropInUICustomization
```

Comment:
Also agreed this is dope! Would there be a way for merchants to supply a custom color scheme in the future? I think it's possible on Android with xml themes, but would have to double-check. Merchants could potentially supply their own color schemes for both light and dark mode.

Reply:
Good question! My thought was that if merchants want to create their own color schemes, they would first specify one of our color schemes (light, dark or dynamic) and then customize additional properties as needed. Theoretically, they could end up overriding every property, so it wouldn't actually matter which color scheme they passed into the initializer.